src/content/reference/web-interface/app-catalog.md

```diff
@@ -0,0 +1,46 @@
++++
+title = "The Giant Swarm App Catalog in The Web Interface"
+description = "What the Giant Swarm App Catalog looks like on our Web Interface and how to use it."
+date = "2019-07-05"
+layout = "subsection"
+weight = 10
++++
+
+# The Giant Swarm App Catalog in The Web Interface
+
+This page will give you an overview of how to do some common tasks related to the
+Giant Swarm App Catalog using our Web Interface.
+
+Go here for an [overview of the Giant Swarm App Catalog](/basics/app-catalog/) instead.
+
+### Viewing all App Catalogs
+
+Our web interface lets you browse the App Catalogs installed on your Control Plane.
+Click on "Apps" in the navigation menu. This link will only be visible if your
+Control Plane has at least one App Catalog installed on it.
+
+The screenshot below shows this page when only the "Helm Stable" and
+"Giant Swarm Incubator" catalogs have been installed:
+
+![app-catalogs](../app-catalogs.png)
+
+### Installing an App
+
+Click on the catalog you'd like to install from. Catalogs have different SLA
```

Comment:
This seems misleading, since to the best of my understanding the only catalog that has SLAs is the managed one.
BTW - SLA = Service Level Agreement, so SLA levels is a redundancy.

Proposed phrasing -
Only apps in the Managed Catalog will be monitored and managed by us. [possible additional verbiage - if we are already talking about the partner catalog]. Apps from other catalogs may have SLAs and guarantees, but those will not be monitored and upheld by Giant Swarm.